### PR TITLE
[Naming] Replace asterisk match with exact compare

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-    public function provideData(): \Iterator
-    {
-        return self::yieldFilesFromDirectory(__DIR__ . '/Fixture');
-    }
-
-    public function provideConfigFilePath(): string
-    {
-        return __DIR__ . '/config/symfony-26.php';
-    }
-}
```
Comment (TomasVotruba) [inline rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc:27]: This was needed in back old days of Rector testing, method name hardcoded :). Not valid anymore and can be removed :+1:

File: rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc [hunk 2]
```diff
@@ -1,27 +0,0 @@
-<?php
-
-namespace Rector\PHPUnit\Tests\PHPUnit60\Rector\ClassMethod\AddDoesNotPerformAssertionToNonAssertingTestRector\Fixture;
-
-class SkipDoTestFileInfoTest extends \Rector\Testing\PHPUnit\AbstractRectorTestCase
```
Comment (samsonasik) [inline rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc:5]: why it removed? it is a fixture file that test skipping `AddDoesNotPerformAssertionToNonAssertingTestRector`

File: rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc (hunk 1 above)
Comment (samsonasik) [reply rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc:27]: this is a fixture file with `.php.inc`, not a class that test a test.

Comment (samsonasik) [reply rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc:27]: oh, I see, `doTestFileInfo` no longer exists 👍

**Pull request merged**